share/examples/bitcoin.conf

```diff
@@ -66,6 +66,10 @@
 # Maximum number of inbound+outbound connections.
 #maxconnections=
 
+#Maximum daily upload target in MiB per day (e.g. 'maxuploadtarget=1024' is 1GiB max throughput per day)
```

Comment:
I'd like `1 GiB` better.

Reply:
@wodry I like it better too. Let's see if I got my squashing and commit correct.